app.js

```diff
@@ -160,7 +160,7 @@ function addEventListener(indicator, emitter, event, f) {
   @param {function(err)} callback - error is undefined if DB exists
 */
 function proceedIfDbValid(db, callback) {
-  db.head('', function(err) {
+  db.info(function(err, data) {
```

Comment:
I know it's not a lot of wasted data, but I think we ought to stick with doing the `HEAD`
```suggestion
  db.server.request({ db: db.config.db, method: 'HEAD' }, function(err) {
```